docs/source/usage/parameters.rst

```diff
@@ -219,8 +219,13 @@ Lattice Elements
 * ``lattice.elements`` (``list of strings``) optional (default: no elements)
     A list of names (one name per lattice element), in the order that they appear in the lattice.
 
+* ``lattice.turns`` (``integer``) optional (default: ``1``)
+    The number of turns through the lattice.
+    Note: this does not check if the lattice is periodic.
```

Comment:
I think this comment is somewhat confusing.  If lattice.turns > 1, then the particles are being tracked through a periodic lattice by definition (all the elements are repeated).  I don't think there's anything that _should_ be checked.

Reply:
Thank you, yes I mean to write that there is no closed orbit check. Will remove, good point.

Reply:
We can also call this `lattice.periods` - depending what is more common lingo...?

Reply:
I don't really know, but I think `lattice.periods` is probably clearer because it does not necessarily suggest bending.